library/src/main/java/com/novoda/downloadmanager/ExternalFilePersistence.java

```diff
@@ -31,7 +31,12 @@ public void initialiseWith(Context context) {
     }
 
     @Override
-    public FilePersistenceResult create(FileName fileName, FileSize fileSize) {
+    public FilePath basePath() {
+        return FilePathCreator.create(getExternalFileDirWithBiggerAvailableSpace().getAbsolutePath(), "/");
```

Comment:
can you extract as constant? also is this a file path separator or a network path separator?

Reply:
It's a file path separator that android seems to deal with `/`

Reply:
we should use the android constant then https://developer.android.com/reference/java/io/File.html#separatorChar